src/index.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { I18nextProvider } from "react-i18next";
 
+import i18n from "./i18n/i18n";
```

Comment:
I'd prefer to have this inside the `App` component. Later on we are going to refactor that component to include the redux store provider, the router, etc... I'd like to have everything in a single place. What do you guys think?

Reply:
Having used i18next extensively, in fact we'll want both this and the I18nextProvider to be in the `App` component. They'll both need information that the store + app have access to.